tools/travis_script.py

```diff
@@ -100,6 +82,4 @@ def get_include_module_option(target_os, target_board):
             if buildtype == 'release':
                 set_release_config_tizenrt()
             exec_docker(DOCKER_TIZENRT_OS_PATH,
-                        ['make', 'IOTJS_ROOT_DIR=../../iotjs',
-                        'IOTJS_BUILD_OPTION=' +
-                        get_include_module_option(target_os, 'artik05x')])
+                        ['make', 'IOTJS_ROOT_DIR=../../iotjs'])
```

Comment:
To test all module, please add `'IOTJS_BUILD_OPTION=--profile=test/profiles/tizenrt.profile'`